### ISSUE TITLE
broken with actual v24.10.0

### ISSUE THREAD

**Issue by SalocinHB** [NikxDa/actual-moneymoney#13]
```
➜  ~ actual-monmon import --logLevel 3
[DEBUG] Connecting to Actual server...
        ↳ Server URL: https://<url>/
[DEBUG] Initializing Actual instance for server https://<url>/ with data directory /Users/nico/.actually/actual-data
[DEBUG] Downloading budget with syncId <syncId>...
[ERROR] Something went wrong trying to download that file, sorry! Visit https://actualbudget.org/contact/ for support. (reason: invalid-schema)
➜  ~
```

**Comment by NikxDa** [comment]
Hi @SalocinHB,

can you please let me know what version of `actual-moneymoney` you are using? I believe there was a breaking change in 24.9.0, but the package version 2.4.0 contained an API update that should have resolved this. This version is working fine for me with 24.10 as well.

I will go ahead and release a new version with the updated API anyway (maybe this should be automated at some point), so that should definitely fix any issues you're seeing :)

**Comment by NikxDa** [comment]
@SalocinHB Latest release (2.5.0) should solve this problem if 2.4.0 didn't already. Let me know if it doesn't and I will reopen this!

**Comment by SalocinHB** [comment]
It was broken in 2.4.0 but fixed in 2.5.0. Thank you!
